lib/src/main/java/ca/barrenechea/widget/recyclerview/decoration/StickyHeaderDecoration.java

```diff
@@ -61,13 +61,24 @@ public StickyHeaderDecoration(StickyHeaderAdapter adapter, boolean renderInline)
     @Override
     public void getItemOffsets(Rect outRect, View view, RecyclerView parent, RecyclerView.State state) {
         int position = parent.getChildAdapterPosition(view);
-
         int headerHeight = 0;
         if (position != RecyclerView.NO_POSITION && hasHeader(position)) {
-            View header = getHeader(parent, position).itemView;
-            headerHeight = getHeaderHeightForLayout(header);
+            final long positionHeaderId = mAdapter.getHeaderId(position);
```

Comment:
What is the purpose of this code? With this code in place, having a header above the first element will not push down that element and the header will overlap parts of it. Reverting it to the old code makes it work as expected.